.github/workflows/run-staging.yaml

```
@@ -0,0 +1,341 @@
+          AWS_REGION: ${{ vars.AWS_REGION }}
+          BACKEND_USE_FARGATE_SPOT: true
+          DOMAIN_NAME: ${{ vars.DOMAIN_NAME }}
+          DJANGO_CONFIGURATION: Staging
+          DJANGO_SETTINGS_MODULE: settings.staging
+          ENABLE_CRON_TASKS: false
+          ENABLE_NAT_GATEWAY: false
+          ENABLE_RDS_PROXY: false
+          ENVIRONMENT: staging
+          FRONTEND_USE_FARGATE_SPOT: true
+          PROJECT_NAME: nest
+          RELEASE_VERSION: ${{ needs.set-release-version.outputs.release_version }}
```

actionlint (1.7.12)

[error] property "set-release-version" is not defined in object type {bootstrap-nest-staging-infrastructure: {outputs: {}; result: string}}

(expression)